app/survey_config/survey_config.py

```diff
@@ -26,6 +27,16 @@ class SurveyConfig:
     account_service_surveys_path: Optional[str] = None
     title_logo: Optional[str] = None
     title_logo_alt: Optional[str] = None
+    contact_us_url: tuple[dict[str, Any]] = (
+        Link(lazy_gettext("Contact us"), f"{base_url}/contact-us/").__dict__,
+    )
+    cookie_settings_url: Optional[str] = f"{base_url}/cookies"
+    privacy_and_data_protection_url: tuple[dict[str, Any]] = (
+        Link(
+            lazy_gettext("Privacy and data protection"),
+            f"{base_url}/privacy-and-data-protection",
+        ).__dict__,
+    )
```

Comment:
This is a dict not tuple. It comes out as a tuple due to the trailing comma. However, that is superseeded by the fact that this should be the string urls, not `Link`, this is already wrong on master, but we should sort it now since it is being updated. We are actually sending a dict to the view as the url, instead of the str. If you generate a 500 error on submission, you can see the contact us link is mangled.

So in the init, we create the string urls which can be made into `Link` object for the footer portion.

Reply:
We should also be consistent with the trailing `/`. Since runner default to including it, we can include for these also.

Reply:
So all the Link dataclasses references are gone from this file, it creates strings now, which business config uses.